From 959b84a852789064f297bc522a12348e97898f41 Mon Sep 17 00:00:00 2001
From: Denys Vlasenko <dvlasenk@redhat.com>
Date: Fri, 29 Mar 2013 16:00:43 +0100
Subject: [LIBREPORT PATCH 2/5] curl upload helper: upload data with
 "application/octet-stream" content type

This fixes a forgotten bit of code where "???" was 'temporarily'
used instead of correct content type.

Signed-off-by: Denys Vlasenko <dvlasenk@redhat.com>
---
 src/lib/curl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib/curl.c b/src/lib/curl.c
index baf6902..d97e4db 100644
--- a/src/lib/curl.c
+++ b/src/lib/curl.c
@@ -600,7 +600,7 @@ char *upload_file(const char *url, const char *filename)
     post_state_t *state = new_post_state(POST_WANT_ERROR_MSG);
     post(state,
                 whole_url,
-                /*content_type:*/ "???",
+                /*content_type:*/ "application/octet-stream",
                 /*additional_headers:*/ NULL,
                 /*data:*/ filename,
                 POST_DATA_FROMFILE_PUT
-- 
1.8.1.4

